src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -1,7 +1,7 @@
 # {{ cephadm_managed }}
 [gateway]
 name = {{ name }}
-group = {{ spec.group }}
+group = {{ spec.group if spec.group is not none else '' }}
```

Comment:
Maybe alternative fix could be to initialize in nvmeof spec constructor group field with empty string ("") instead of None

Reply:
@baum I will check and maybe create another PR.